src/services

```diff
@@ -90,8 +90,9 @@ p({"class":"foam.nanos.boot.NSpec", "name":"ruleHistoryDAO",                   "
 p({"class":"foam.nanos.boot.NSpec","name":"emailConfig",                       "lazy":false, "serve":false, "service":{"class":"foam.nanos.app.EmailConfig"}})
 p({"class":"foam.nanos.boot.NSpec", "name":"serializationTestEchoService", "boxClass": "foam.nanos.test.EchoServiceSkeleton", "lazy":true, "serve":true, "authenticate": true, "serviceClass":"foam.nanos.test.SerializationTestEchoService", "client":"{\"class\":\"foam.nanos.test.ClientEchoService\",\"delegate\":{\"class\":\"foam.box.SessionClientBox\",\"delegate\":{\"class\":\"foam.box.HTTPBox\",\"method\":\"POST\",\"url\":\"service/serializationTestEchoService\"}}}"})
 p({"class":"foam.nanos.boot.NSpec","name":"themeDAO","description":"Stores Themes, which are used to style the application.","serve":true,"authenticate":false,"serviceScript":"return new foam.dao.EasyDAO.Builder(x)\u000a  .setAuthenticateRead(false)\u000a  .setAuthenticate(true)\u000a  .setPm(true)\u000a  .setSeqNo(true)\u000a  .setOf(foam.nanos.theme.Theme.getOwnClassInfo())\u000a  .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)\u000a  .setJournalName(\"themes\")\u000a  .build();","client":"{\"of\":\"foam.nanos.theme.Theme\"}","authNotes":"Needs to be unauthenticated because we need to be able to theme apps before the user has logged in. However, we still need to restrict write access, which we do via an AuthorizationDAO."})
+
 p({"class":"foam.nanos.boot.NSpec","name":"localGroupPermissionJunctionDAO","description":"The junction DAO for the many-to-many relationship between groups and permissions","serve":false,"serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.auth.GroupPermissionJunction.getOwnClassInfo()).setAuthenticate(false).setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setJournalName(\"groupPermissionJunctions\").setPm(true).setDecorator(new foam.nanos.auth.AuthorizationDAO(x)).build()"})
-p({"class":"foam.nanos.boot.NSpec","name":"groupPermissionJunctionDAO","description":"The junction DAO for the many-to-many relationship between groups and permissions","serve":true,"serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.auth.GroupPermissionJunction.getOwnClassInfo()).setInnerDAO((foam.dao.DAO) x.get(\"\")).build();","client":"{\"of\": \"foam.nanos.auth.GroupPermissionJunction\"}"})
+p({"class":"foam.nanos.boot.NSpec","name":"groupPermissionJunctionDAO","description":"The junction DAO for the many-to-many relationship between groups and permissions","serve":true,"serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.auth.GroupPermissionJunction.getOwnClassInfo()).setAuthenticate(true).setInnerDAO((foam.dao.DAO) x.get(\"localGroupPermissionJunctionDAO\")).build();","client":"{\"of\": \"foam.nanos.auth.GroupPermissionJunction\"}"})
```

Comment:
We don't need `setAuthenticate(true)`. The `GroupPermissionJunction` model implements the `Authorizable` interface. `groupPermissionJunctionDAO` should be wrapped in an `AuthorizationDAO`.